lib/dartino.dart

```diff
@@ -176,3 +180,14 @@ _runAppOnDevice(event) async {
     atom.notifications.addInfo('Launched app on device', dismissable: true);
   }
 }
+
+_showGettingStarted(event) async {
+  shell.openExternal('https://dartino.org/index.html');
+}
+
+_showSdkDocs(event) async {
+  Sdk sdk = await findSdk(null);
+  if (sdk == null) return;
+  // TODO(danrubel) convert Windows file path to URI
+  shell.openExternal('file://${sdk.sdkRootPath}/docs/index.html');
```

Comment:
You may want to look at `Uri.file(String path, {bool windows})` here (but I'd verify first that it does what you want).